tasklist/client/src/Tasks/CollapsiblePanel/index.tsx

```diff
@@ -302,19 +302,14 @@ const CollapsiblePanel: React.FC = () => {
         </ul>
       </nav>
       {filtersModal}
-      <Modal
-        danger
-        open={customFilterToDelete !== null}
-        size="xs"
-        modalLabel="Delete filter"
-        modalHeading="The custom filter will be deleted."
-        primaryButtonText="Confirm deletion"
-        secondaryButtonText="Cancel"
-        onRequestClose={() => {
+      <DeleteFilterModal
+        data-testid="direct-delete-filter-modal"
+        filterName={customFilterToDelete ?? ''}
+        isOpen={customFilterToDelete !== undefined}
```

Comment:
```suggestion
        isOpen={customFilterToDelete !== null}
```

shouldn't this be a null check?

Reply:
Yep, it was a mistake because on the next PR I changed it to `undefined`

That was also the reason the visual regression test was failing